AzureCalling/Controllers/CallViewController.swift

```diff
@@ -312,6 +311,10 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
         infoHeaderView.toggleDisplay()
     }
 
+    @IBAction func switchCamera(_ sender: UIButton) {
```

Comment:
This is one way I figured to invoke callingContext's method.
We also need to enable this feature in lobbyView as the next work item.

Reply:
Let's use closure/delegate to pass the action from participant view rather than direct linking participant view's button to callingView. 
Please correct me if you have any good reference resource with better approach

Reply:
Closure is perfectly fine in this scenario.